src/main/java/at/hannibal2/skyhanni/features/misc/PatcherSendCoordinates.kt

```diff
@@ -30,10 +30,12 @@ object PatcherSendCoordinates {
 
     /**
      * REGEX-TEST: hannibal2: x: 2, y: 3, z: 4
+     * REGEX-TEST: hannibal2: x: 2, y: 3, z: 4broken
+     * REGEX-TEST: hannibal2: x: 2, y: 3, z: 4 extra text
      */
     private val coordinatePattern by RepoPattern.pattern(
         "misc.patchercoords.coords",
-        "(?<playerName>.*): [xX]: (?<x>[0-9.-]+),? [yY]: (?<y>[0-9.-]+),? [zZ]: (?<z>.*)"
+        "(?<playerName>.*): [xX]: (?<x>[0-9.-]+),? [yY]: (?<y>[0-9.-]+),? [zZ]: (?<z>[0-9.-]+(?: .*)?)"
```

Comment:
```suggestion
        "(?<playerName>.*): [xX]: (?<x>[0-9.-]+),? [yY]: (?<y>[0-9.-]+),? [zZ]: (?<z>[0-9.-]+)(?:.*)"
```

why not just
![image](https://github.com/user-attachments/assets/78064334-7eb6-472e-82d5-da88b2e0b345)

Reply:
The second line in the regex test will break SkyHanni code. We expect one empty line between z and extra text.
In a future pr we might want to change the SkyHanni code, but for now, we want to also fix this for older sh versions via a regex only change.